src/modules/vtol_att_control/vtol_type.h

```diff
@@ -260,18 +261,11 @@ class VtolType
 	 */
 	bool set_idle_fw();
 
+	void set_all_motor_state(const motor_state target_state, const int value = 0);
 
-	/**
-	 * @brief      Sets state of a selection of motors, see struct motor_state
-	 *
-	 * @param[in]  current_state  The current motor state
-	 * @param[in]  next_state     The next state
-	 * @param[in]  value          Desired pwm value if next_state =
-	 *                            motor_state::VALUE
-	 *
-	 * @return     next_state if succesfull, otherwise current_state
-	 */
-	motor_state set_motor_state(const motor_state current_state, const motor_state next_state, const int value = 0);
+	void set_main_motor_state(const motor_state target_state, const int value = 0);
+
+	void set_alternate_motor_state(const motor_state target_state, const int value = 0);
```

Comment:
It's cleaner to remove the `const` in the declaration as they are not useful for the "user" of the function (passing a copy, there is no risk to change the value). However, you can keep it in the definition to avoid changing the local copy.
```suggestion
	void set_main_motor_state(motor_state target_state, int value = 0);

	void set_alternate_motor_state(motor_state target_state, int value = 0);
```
Same for all the other function not receiving pointers or references

Reply:
@bresch Addressed here: https://github.com/PX4/PX4-Autopilot/pull/16356/commits/062f7955411583824c295a8bda5b5f6fb6ac4f0b